corpus/config/models.py

```diff
@@ -61,6 +79,7 @@ class Society(models.Model):
         verbose_name="Dark Image", upload_to="img/logo/", blank=True, null=True
     )
     description = models.TextField(verbose_name="Description")
+    sig = models.IntegerField(verbose_name="SIG ID", default=-1, null=True)
```

Comment:
Use a relation and not just an `IntegerField`. One society can be part of multiple SIGs, and one SIG can have multiple societies, so look into what field accurately represents this relationship.

Reply:
done